torch_xla/core/xla_model.py

```diff
@@ -238,6 +238,11 @@ def xla_device(n=None, devkind=None):
   Returns:
     A `torch.device` with the requested instance.
   """
+  if xu.check_env_flag('XLA_USE_SPMD'):
```

Comment:
Let's also add some comment here, explaining why this override is needed.

Reply:
Done